src/docs/background/payments-overview/index.md

```diff
@@ -0,0 +1,446 @@
+---
+title: Payments in Flex
+slug: payments-overview
+updated: 2021-11-18
+category: background
+ingress:
+  This article introduces how payments work in Flex in general, and
+  describes the default Stripe payment gateway integration
+published: false
```

Comment:
🤔 should this be `published: true` or did I miss something?

Reply:
You're right, I have no idea why it was left as `false`. Toggled!